dali/kernels/signal/dct/dct_gpu.cu

```diff
@@ -29,11 +29,49 @@ namespace kernels {
 namespace signal {
 namespace dct {
 
+struct Lifter {
+  DALI_HOST_DEV
+  explicit Lifter(float coeff): coeff_(coeff) {}
+
+  DALI_HOST_DEV
+  float operator()(float val) { return val * coeff_; }
+
+  const float coeff_;
+};
+
+struct IdLifter {
```

Comment:
Why not just use `identity` from "core/util.h"?

Reply:
I've remove LifterTabble